master.go

```diff
@@ -108,19 +116,21 @@ func (m *Master) ScheduleFromId(ID string) error {
 }
 
 func (m *Master) logScheduleInfo(message string, payload *MetreMessage) {
-	log.Info(fmt.Sprintf("metre.master.Schedule: %s - taskId:%s, uid:%s, jobGUID:%s",
-		message,
-		payload.TaskId,
-		payload.UID,
-		payload.JobGUID))
+	logging.Logger.Info("metre master scheduling info",
```

Comment:
Since we're changing the way the message is written, then I'd prefer we log the namespace as a zap.String...the whole point was troubleshooting and easy grep.